src/main/java/de/tum/cit/aet/artemis/programming/service/localci/LocalCITriggerService.java

```diff
@@ -105,13 +109,18 @@ public class LocalCITriggerService implements ContinuousIntegrationTriggerServic
 
     private final ProgrammingExerciseBuildConfigService programmingExerciseBuildConfigService;
 
+    private static final int DEFAULT_BUILD_DURATION = 17;
+
+    private static final double BUILD_DURATION_SAFETY_FACTOR = 1.2;
```

Comment:
What is the rationale behind 1.2? 
I think it is always good to document something like this (does not need to be changed in this PR)